iOS/DuckDuckGo/en.lproj/Localizable.strings

```diff
@@ -1926,9 +1926,6 @@ https://duckduckgo.com/mac";
 /* Text shown on the favorites info tooltip */
 "new.tab.page.tooltip.body" = "On any site, open the ••• menu and select **Add Favorite** to add it to your new tab page.";
 
-/* Do not translate - stringsdict entry */
-"number.of.tabs" = "number.of.tabs";
```

Comment:
This shouldn't have an impact because the actual translation was in the `stringsdict`